packages/bluefish.rb

```diff
@@ -29,18 +16,25 @@ class Bluefish < Package
   depends_on 'python3'
   depends_on 'shared_mime_info'
   depends_on 'xdg_base'
-  depends_on 'sommelier'
+  depends_on 'wayland_protocols'
+  depends_on 'mesa'
+  depends_on 'xcb_util'
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-maintainer-mode'
+    system "./configure #{CREW_OPTIONS}"
     system "make"
   end
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    FileUtils.mv '#{CREW_DEST_PREFIX}/bin/bluefish', '#{CREW_DEST_PREFIX}/bin/bluefish_orig'
+    system "cat <<'EOF'> bluefish
+WAYLAND_DISPLAY=wayland-0
+GDK_BACKEND=wayland
+DISPLAY=
```

Comment:
I wonder if we can put these variables in a `.env` file instead?

Reply:
Can we put the `env` file into `install.sh`?

Reply:
We might be able to do something without the need of the shell script.